cmd/server/main.go

```diff
@@ -28,15 +28,20 @@ func main() {
 			Usage: "ocid pod sandbox dir",
 		},
 		cli.StringFlag{
-			Name:  "runtime",
-			Value: "/usr/bin/runc",
-			Usage: "OCI runtime path",
-		},
-		cli.StringFlag{
 			Name:  "containerdir",
 			Value: "/var/lib/ocid/containers",
 			Usage: "ocid container dir",
 		},
+		cli.StringFlag{
+			Name:  "socket",
+			Value: unixDomainSocket,
+			Usage: "path to ocid socket",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Value: "/usr/bin/runc",
```

Comment:
> Can we make this /usr/sbin/runc which is the "default" install location for runC?

it installs by default on `/usr/local/sbin/runc` - from upstream repo https://github.com/opencontainers/runc/blob/master/Makefile#L6 (at least on my system). 
> Or do other distributions put this in /usr/bin?

Fedora ships runc under `/usr/bin/runc` also.

If you'd like to change this feel free to open a PR @cyphar - I'm not changing the runc path in this PR (just moving it up and down).